ui/i18n/src/main/res/values/strings.xml

```diff
@@ -52,11 +52,11 @@
     <string name="statistics_counting_total">Played in total</string>
 
     <!-- Home fragment -->
-    <string name="home_surprise_title">Get surprised</string>
```

Comment:
For the Home screen I would keep sentence case for now. It's less obtrusive (which is good for large lists of settings, I think it fits less in the actual interface).